src/SqlFormatter.php

```diff
@@ -1045,7 +1045,7 @@ private static function getNextToken(string $string, ?array $previous = null) :
         ];
     }
 
-    private static function getQuotedString(string $string) : string
+    private static function getQuotedString(string $string) : ?string
```

Comment:
I'm curious if this is hiding a long-standing bug, and `$ret` should be initialized as `$string`, so the return type will stay `string` (not nullable)

Reply:
Just from reading the signature it seems you're right, I will look into it!

Reply:
Do you like it better like this?